docs/user-research/MO-user-groups-feedback.md

```diff
@@ -0,0 +1,276 @@
+  various improvements to the command line].
+
+* RR: When you interact with a suite via the UI, it would be very useful to
+  see the corresponding commands, e.g. in command logs (obviously there will be
+  no command equivalent to the action of 'adding in views' etc., but for
+  anything with a suite command equivalent). [We agreed that it would indeed be
+  useful & we would note it, but also described how back-end changes mean that
+  the UI & command line will be synchronised so that most/all commands
+  correspond to UI actions & vice versa, & the command equivalents should be
+  mostly evident.]
+
+* AA: How will this all work for shared accounts? [We explained we are still
+  planning what to do with shared accounts, but probably the shared account
+  owner will grant various levels of access (e.g. limited control, read-only,
+  etc.) to sets of users as desired, such that when logged-in to the Cylc Hub
+  as themselves, users will be able to interact, to the permitted extent, with
+  any shared accounts which owners have allowed them to.]
+
+* MT: What are those panels containing the views? [We elaborated on
+  the multi-view system in the design mock-up.]
+
+* MB: Will you still be able to have multiple instances open, like multiple
+  windows for the old GUIs? I guess it would be multiple tabs now? [We
+  confirmed that multiple tabs would be the browser equivalent, & that the UI
+  will work across mutliple tabs, but explained how we are trying to reduce
+  the need for multiple (and especially large numbers of) tabs with the
+  multi-view panelling system, which aims to show users as much information
+  as they could want to see, all within a single tab.
+
+* AD: We have set up a feedback page under the retooling project pages & we'll
+  keep putting recommendations on there [We said thanks.].
+
+* MB: In general, Cylc meets our needs, but Rose does not. [We said this was
+  noted.]
+
+
+#### From feedback pages
+
+Notable points (relating to the Cylc UI) from feedback pages:
+
+* [From the agenda notes:] "TSUG noted that this [the Rose replacement] will
+  be of most benefit to users represented here. The *Cylc upgrades are nice
+  to have* but the Rose upgrades will be vital to allow us to get the most
+  out of our trials."
+
+NB: those pages contain quite a lot of useful feedback, but not much
+(just the above) relating to the Cylc UI, mainly to Rose config-edit, plus one
+non-UI Cylc matter (which is of relevance to future Python API work):
+
+* Under the section 'User requirements for new suite control system', from MB:
+  "Better support for determining more complex aspects of the graph of a
+  suite. Some aspects of the suite graph of an ideal version of an evaluation
+  suite would be sufficiently complex that using jinja2 is unwieldy, limiting
+  and difficult to maintain. It would be useful to have the ability to define
+  the suite programmatically either as an initial app, or by some other means.
```

Comment:
Bring on Cylc 9!

Reply:
Absolutely. It was nice to read that & think "we've got you covered there" (in our intentions & motivations for them, at least!)